composer.json

```diff
@@ -15,7 +15,8 @@
         "nuwave/lighthouse": "^4.6",
         "barryvdh/laravel-cors": "^0.11.3",
         "guzzlehttp/guzzle": "~6.0",
-        "geoip2/geoip2": "~2.0"
+        "geoip2/geoip2": "~2.0",
+        "stripe/stripe-php": "^7.14"
```

Comment:
@AlexeiaW think you may have put this one in to experiment with Stripe. Let's leave it out for now though and can be part of a Stripe PR.

Reply:
removed the package